src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js

```diff
@@ -227,17 +227,15 @@ const CourseSessionStatus = () => {
                                         id,
                                         course_id,
                                         instructor,
-                                    } = sessionDataParse(session);
+                                    } = 
+                                    
+                                    sessionDataParse(session);
```

Comment:
fix spacing

Reply:
This has been done, will be included when test is finished and pull request is updated.